pyccel/ast/builtin_methods/set_methods.py

```diff
@@ -102,7 +101,6 @@ class SetClear(SetMethod):
     def __init__(self, set_variable):
         super().__init__(set_variable)
 
-
 class SetCopy(SetMethod):
```

Comment:
Any reason for this anti-PEP change? https://peps.python.org/pep-0008/#blank-lines

Reply:
Fixed : 55d1560